awesome/src/lua/mod.rs

```diff
@@ -52,12 +37,26 @@ thread_local! {
 ///
 /// Best way to help out: comment out one of these lines, fix what breaks.
 fn load_shims(lua: &Lua) {
+    let shims_path: PathBuf = std::env::current_exe().unwrap().parent()
+        .ok_or(()).unwrap().join("../../tests/awesome/tests/examples/shims/");
```

Comment:
Why did you put `ok_or(())` here? This drops important error information for no benefit.

Reply:
Sorry, I'm not very versed in Rust yet. What should I've used instead?

Reply:
Just remove the `ok_or(())` call.